src/main/scala/rps/Game.scala

```diff
@@ -0,0 +1,61 @@
+package rps
+
+object Move extends Enumeration {
+  type Move = Value
+  val Rock = Value("0")
+  val Paper = Value("1")
+  val Scissor = Value("2")
+
+  def isOrderType(s: String) = values.exists(_.toString == s)
+}
+
+object Result extends Enumeration {
+  type Result = Value
+  val Won = Value("you won!")
+  val Lost = Value("you lost...")
+  val Tie = Value("it's a tie!")
+}
+
+object Game {
+  def getRandomMove(): Move.Move = {
+    Move(scala.util.Random.nextInt(Move.maxId))
+  }
+
+  def getMoveName(move: Move.Move): String = move match {
+    case (Move.Rock)    => "Rock"
+    case (Move.Paper)   => "Paper"
+    case (Move.Scissor) => "Scissor"
+  }
+
+  def getGameResult(a: Move.Move, b: Move.Move): Result.Result = {
+    if (a == b) {
+      Result.Tie
+    } else
+      (a, b) match {
+        case (Move.Rock, Move.Paper)    => Result.Won
+        case (Move.Paper, Move.Scissor) => Result.Won
+        case (Move.Scissor, Move.Rock)  => Result.Won
+        case _                          => Result.Lost
+      }
+  }
```

Comment:
good use of pattern matching 👍 
You don't need that `if`, try including the `Tie` case in the pattern matching, for extra readability.

Also, you don't need three different clauses for `Result.Won`, you can use `|` to match on multiple cases.